app/src/main/java/de/westnordost/streetcomplete/quests/address/AddHousenumberForm.kt

```diff
@@ -67,16 +78,22 @@ class AddHousenumberForm : AbstractQuestFormAnswerFragment<HousenumberAnswer>()
         }
     }
 
-    override fun isFormComplete() = (!isShowingHouseNumberHint || isHousename) && createAnswer() != null
+    override fun isFormComplete() = (!isShowingHouseNumberHint || interfaceMode == InterfaceMode.HOUSENAME ) && createAnswer() != null
```

Comment:
```suggestion
    override fun isFormComplete() = (!isShowingHouseNumberHint || interfaceMode == InterfaceMode.HOUSENAME) && createAnswer() != null
```

Reply:
Also, this will still show the OK button in the `HOUSENUMBER_AND_HOUSENAME` when only one input field contains text, right?

Reply:
I forgot to push :(